test/test_interfaces.py

```diff
@@ -78,3 +80,33 @@ def test_is_boolean_network(self):
 
         not_bool_net = self.MultipleBaseNetwork()
         self.assertFalse(is_boolean_network(not_bool_net))
+
+    def test_neighbors_ECA(self):
+        eca = ca.ECA(30)
+
+        self.assertTrue(neighbors(eca,size=4),[set([0, 1, 3]), 
+                                               set([0, 1, 2]), 
+                                               set([1, 2, 3]), 
+                                               set([0, 2, 3])])
+
+        with self.assertRaises(AttributeError):
+            neighbors(eca)
+
+    def test_neighbors_WTNetwork(self):
+        net = bnet.WTNetwork([[1]])
+
+        self.assertTrue(neighbors(net),[set([0])])
+
+    def test_neighbors_LogicNetwork(self):
+        net = bnet.LogicNetwork([((0,), {'0'})])
+
+        self.assertTrue(neighbors(net),[set([0])])
+
+    def test_neighbors_no_attribute(self):
```

Comment:
All this looks good!  Maybe too good, in the sense that I feel like I want to require that _all_ neet networks have a neighbors attribute.  Did we decide against this?

Reply:
Oh okay, cool.  I was just confused by the "IsNetwork" object.  If a network without neighbors doesn't qualify as a neet network, should we give this to the IsNetwork object?

Reply:
I think what we want is to modify `neet.interfaces.is_network` to require that there be a `neighbors` method. Then this check should raise a `TypeError` since `net` would be in invalid type.

Reply:
@dglmoore @bcdaniels Ok I just pushed some updates that should enforce this now